tests/e2e/vLLM/recipes/INT8/w8a8_static_asym.yaml

```diff
@@ -0,0 +1,7 @@
+cadence: "nightly"
+test_type: "regression"
+model: TinyLlama/TinyLlama-1.1B-Chat-v1.0
+scheme: W8A8
+dataset_id: HuggingFaceH4/ultrachat_200k
+dataset_split: train_sft
+recipe: tests/e2e/vLLM/recipes/INT8/recipe_w8a8_static_asym.yaml
```

Comment:
Can you add a saving path to these files?
`save_dir: TinyLlama-1.1B-Chat-v1.0-W8A8-Static-Asym`